tricat_211/ObstaclePoint[] obstacle
